src/main/java/teammates/common/util/Url.java

```diff
@@ -130,11 +81,11 @@ public String toString() {
     }
```

Comment:
If not in this issue, we can do it as a separate issue. It's a technical debt we have to pay off one way or other.
It's fine to omit the method we don't need though.